benefits/templates/registration/logged_out.html

```diff
@@ -2,7 +2,7 @@
 {% load static %}
 
 {% block title %}
-    Logged out | Cal-ITP Benefits Administrator
+    Logged out | {{ site_title }}
 {% endblock title %}
```

Comment:
This `logged_out` page extends from `agency-base`, and _not_ the default Django admin log out template, so it's necessary to explicitly spell out `Logged out`